lib/ansible/modules/windows/setup.ps1

```diff
@@ -226,7 +226,7 @@ if($gather_subset.Contains('env')) {
 if($gather_subset.Contains('facter')) {
     # See if Facter is on the System Path
     Try {
-        $facter_exe = Get-Command facter -ErrorAction Stop
+        Get-Command facter -ErrorAction Stop
```

Comment:
This will cause the output of `Get-Command` to appear in the stdout returned output to Ansible and could make it fail to parse the json string that is also there. If you wish to discard the output just do `Get-Command facter -ErrorAction Stop > $null`.